README.adoc

```diff
@@ -33,10 +33,10 @@ You will learn how to use the Java Persistence API (JPA) to map Java objects to
 perform create, read, update and delete (CRUD) operations in your microservices. 
 
 JPA is a Java specification for representing relational database table data as Plain Old Java Objects (POJO).
-The JPA specification is implemented by several different persistence providers. In this guide you will 
-use EclipseLink as the persistence provider because it is the implementation that is shipped with Liberty.
+An implementation of the JPA specification is referred to as a persistence provider. In this guide you will 
+use EclipseLink as the persistence provider because it is one the implementations shipped with Liberty.
```

Comment:
Maybe more like "it is the JPA implementation provided by the jpa-2.2 feature" ?